metrics.yaml

```diff
@@ -46,3 +47,156 @@ enrollment:
     notification_emails:
       - than@mozilla.com
     expires: never
+
+user:
```

Comment:
Hey @hamilton , do you want to take a first pass at the labels here? (you can see the rendered doc too)